backend/src/main/java/com/roc/app/user/match/UserMatchController.java

```diff
@@ -0,0 +1,22 @@
+package com.roc.app.user.match;
+
+import com.roc.app.user.match.dto.UserMatchDto;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/users/{userId}/matches")
+public class UserMatchController {
```

Comment:
We have `Match` in model and so here we should have just `MatchController`. Same applies to service and repository.
```suggestion
public class MatchController {
```